packages/core/src/lib/features/LocationDelegationFeature.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import {EmptyFeature} from './EmptyFeature';
+
+export type LocationDelegationConfig = {
+}
+
+export class LocationDelegationFeature implements EmptyFeature {
```

Comment:
Can we make this class look like `FirstRunFlagFeature` or `AppsFlyerFeature`? They both extend `EmptyFeature` but instead of defining everything they just set the things that are non-empty in the constructor.